js-peer/src/pages/index.tsx

```diff
@@ -3,11 +3,10 @@ import Nav from '@/components/nav'
 import ChatContainer from '@/components/chat'
 import ConnectionPanel from '@/components/connection-panel'
 import { useState } from 'react'
-import { useLibp2pContext } from '@/context/ctx'
 import ConnectionInfoButton from '@/components/connection-info-button'
 
 export default function Chat() {
-  const [isConnectionPanelOpen, setIsConnectionPanelOpen] = useState(false)
+  const [isConnectionPanelOpen, setIsConnectionPanelOpen] = useState(true)
```

Comment:
I don't think this is a good idea. 
```suggestion
  const [isPanelOpen, setisPanelOpen] = useState(false)
```